### ISSUE TITLE
Clean up current QA demo Python and JavaScript code from repo

### ISSUE THREAD

**Issue by lintool** [castorini/Anserini#243]
Per #239 the QA demo code (Python and JavaScript) should be moved out of Anserini, e.g.,

https://github.com/castorini/Anserini/tree/master/src/main/js
https://github.com/castorini/Anserini/tree/master/src/main/python

**Comment by tuzhucheng** [comment]
Which repo should they be moved to, or a new repo?

**Comment by lintool** [comment]
Your call - I suppose the options are Castor, or another "app" repo. Does it make sense to keep Castor pure DL models? Thoughts?

**Comment by tuzhucheng** [comment]
Separate repo makes sense. Castor and Anserini are core IR/DL repos and the demos use these repos to build an application.

**Comment by lintool** [comment]
Okay, so our general layout will be like this?

```
castorini/
  |--- Castor/
  |--- Castor-data/
  |--- Anserini/
  |--- app1/
  |--- app2/
...
```

If we agree on this layout, then we can hard code relative paths all over.
Thoughts?

**Comment by tuzhucheng** [comment]
Yep. Currently we already do this for Castor-data and Castor, but the relative paths should just be defaults in configuration files or command line args.

**Comment by tuzhucheng** [comment]
Likewise should https://github.com/castorini/Castor/tree/master/anserini_dependency be moved into the app repo as well? There is a copy of the demo in here worked on @rosequ and @MeowFei.

**Comment by tuzhucheng** [comment]
Fully resolved by #267
